Jenkinsfile

```diff
@@ -1,15 +1,72 @@
 #!/usr/bin/env groovy
 
-properties([
-    parameters([
-        string(defaultValue: 'ci40', description: 'OpenWrt branch to test against', \
-            name: "OPENWRT_BRANCH")
-    ])
-])
-
-stage('Trigger openwrt build') {
-    build job: "../openwrt/${params.OPENWRT_BRANCH}", parameters: [
-        string(name: 'OVERRIDE_CREATOR', value: "${BRANCH_NAME}")
-    ]
+/**
+ * Build Creator feed packages
+ *
+ * Configure as needed, pull in the feeds and run the build.
+ * Overrides Creator feed with local clone.
+ *
+ */
+
+node('docker && imgtec') {
+    def docker_image
+    stage('Prepare Docker container') {
+        docker_image = docker.image "imgtec/creator-builder:latest"
+    }
+    docker_image.inside {
+//      deleteDir() // Wipe the workspace so we are building completely clean
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                    [$class: "RelativeTargetDirectory", relativeTargetDir: "${WORKSPACE}/Creator-feed"],
+                ],
+            ])
+
+            // Download and extract Creator SDK
+            sh "wget -O ${WORKSPACE}/sdk.tar.bz2 \
+                http://downloads.creatordev.io/creator-openwrt/v1/pistachio/marduk/creator-sdk-1.tar.bz2 && \
+                tar -xf ${WORKSPACE}/sdk.tar.bz2"
+
+            // Enable signed package indexing
+            sh "echo '\nconfig SIGNED_PACKAGES\n\tbool\n\tdefault y\n' >> ${WORKSPACE}/creator-sdk/Config.in"
```

Comment:
There is no need for `${workspace}` in all these commands as its all relatvie to where you currently are anyway